scripts/linux/buildAPIProxy.sh

```diff
@@ -0,0 +1,202 @@
+#!/bin/bash
+#https://github.com/Azure/iotedge/blob/iiot/scripts/linux/cross-platform-rust-build.sh
+###############################################################################
+# This script builds a static binary of the api-proxy-module
+###############################################################################
+
+set -e
+
+###############################################################################
+# Define Environment Variables
+###############################################################################
+# Get directory of running script
+ARCH=$(uname -m)
+DIR=$(cd "$(dirname "$0")" && pwd)
+TARGET=
+STRIP=
+SCRIPT_NAME=$(basename "$0")
+PROJECT=
```

Comment:
why project is a variable?  This script is only used for build api proxy and the project should be fixed or at least have default value, right?

Reply:
I can remove it but again it will make the merge between buildAPIProxy.sh and buildimage.sh more difficult.

Reply:
Same for DOCKERFILE and DOCKER_IMAGENAME.

Reply:
That is a potential tweaking parameter. I'll keep it as is. I however removed the unused variable DOCKERFILE and DOCKERIMAGENAME